code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -17,7 +17,7 @@
 /obj/item/reagent_containers/blood/on_reagent_change(changetype)
 	if(reagents)
 		var/datum/reagent/blood/B = reagents.has_reagent(/datum/reagent/blood)
-		if(B && B.data && B.data["blood_type"])
+		if(B?.data && B.data["blood_type"])
```

Comment:
This is runtiming and causing travis to fail.

Reply:
Yeah it looks like someone badly coded the has_reagent proc. I've fixed it.